### DISCUSSION TITLE
Option to disable configurations

### DISCUSSION THREAD

**Discussion by philmassie** [davidrapan/ha-solarman#304] [Ideas]
I think this wont be a popular idea, but I would love an option to disable all possible configuration updates from Home Assistant.
I always get a little scared I'll accidentally swipe a dangerous setting when I scroll through all the configs in device settings. I'd like the option to just disable them.

I appreciate that a huge amount of effort has gone into building them, and don't mean to sound ungrateful, apologies if it comes across that way.
Thanks
Phil

**Comment by davidrapan** [comment]
I have this on my mind for quite some time now but still did not figured out how to accomplish this efficiently.

But it will be done in near future. 😉
